packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -40,7 +46,7 @@ describe('HttpServer (integration)', () => {
   });
 
   it('exports original port', async () => {
-    server = new HttpServer(dummyRequestHandler, {port: 0});
+    server = new HttpServer(dummyRequestHandler, {port: 0} as HttpOptions);
```

Comment:
Why is this explicit cast necessary? Ideally, the typings should allows consumers to provide `{port: 0}` config without any explicit type casts. 

Maybe we need to fix the definition of `HttpOptions` as follows?

```ts
export interface HttpOptions extends ListenerOptions {
  protocol?: 'http';
}
```